Add gateway claiming reference (#295)
TheThingsIndustries/lorawan-stack-docs: doc/content/gateways/gateway-claiming/claim-gateways.md

---
title: "Claim a Gateway"
description: ""
weight: 2
distributions: ["Enterprise", "Cloud"]
--- 

> You can update the example below with your tenant ID and cluster ID by filling them here.

Reviewer: ```suggestion
This section provides an example on how to claim your gateway. You can update the example with your tenant ID and cluster ID by providing your values below.
```

> {{< tenant-cluster-selector >}}

Reviewer 2: Add CLI only shortcode here

> 

Reviewer: ```suggestion

{{< cli-only >}}

```

## Prerequisites

> 1. An account in The Things Industries Cloud.
> 2. EUI of the gateway that you wish to claim.
> 3. The Claim Authentication Code for the gateway (shared out of band by the current gateway owner).
> 4. The Current CUPS Key used by the gateway (for {{% lbs %}} only).

Reviewer: ```suggestion
1. A user account in The Things Industries Cloud
2. EUI of the gateway that you wish to claim
3. The Claim Authentication Code for the gateway (shared out of band by the current gateway owner)
4. The current CUPS Key used by the gateway (for {{% lbs %}} only)
```

Reviewer: Why The Things Industries Cloud? In other file you're using The Things Stack Cloud, but I'm confused with that too since labels tell The Things Stack Cloud and Enterprise. Please be consistent with these (or tell me if I'm misunderstanding something)

Author: Yeah you're right. This is corrected.

> ## General

Reviewer: ```suggestion
## Note on Using the Gateway ID
```

Author: This is an `H2` section that contains 2 H3 sections for LBS and non-LBS so I don't think `Note on Using the Gateway ID` is the correct heading.

Reviewer: Ok I get it. Although it makes more sense to change other sections to H2, because they contain the most important info, and put this as a note. Doesn't really make sense to start with explaining the flags that are optional

> The `--target-gateway-id` field is optional. If left blank, the server will attempt to use `eui-<gateway-id>` as the gateway ID. If a gateway already exists with either Gateway ID values, claiming will fail.

Reviewer: ```suggestion
For the commands below, the `--target-gateway-id` field is optional. If left blank, the server will attempt to use `eui-<gateway-id>` as the gateway ID. If a gateway already exists with either Gateway ID values, claiming will fail.
```

If the gateway has been successfully claimed, so you can return to the Console and you will see it in your list of gateways.

Once a gateway is claimed, it needs to restarted for the new changes to take effect.

### Claim a Gateway (LoRa Basics Station)

> If the gateway is a {{% lbs %}} Gateway and has a CUPS server configured to a The Things Industries Cloud tenant, then you must additionally setup CUPS redirection. This means that when the newly claimed gateway connects to the CUPS server, the server will update the CUPS endpoint and credentials to the ones of the new owner.

Reviewer: ```suggestion
If the gateway is a {{% lbs %}} gateway and has a CUPS server configured to a The Things Industries Cloud tenant, then you must additionally setup CUPS redirection. This means that when the newly claimed gateway connects to the CUPS server, the server will update the CUPS endpoint and credentials to the ones of the new owner.
```

> If the gateway is a {{% lbs %}} Gateway and has a CUPS server configured to a The Things Industries Cloud tenant, then you must additionally setup CUPS redirection. This means that when the newly claimed gateway connects to the CUPS server, the server will update the CUPS endpoint and credentials to the ones of the new owner.
> 

Reviewer: ```suggestion
If the gateway is a {{% lbs %}} Gateway and has a CUPS server configured to a The Things Industries Cloud tenant, then you must additionally set up the CUPS redirection. This means that when the newly claimed gateway connects to the CUPS server, the server will update the CUPS endpoint and credentials to the ones of the new owner.

```

The additional fields that need to be set are

> - `--target-cups-uri`: The URI of the CUPS server once the gateway is claimed. For The Things Industries Cloud, this would be;

Reviewer: ```suggestion
- `--target-cups-uri`: The URI of the CUPS server once the gateway is claimed. For The Things Industries Cloud, this would be:
```

    <p>
    <code data-content="cluster-address">
    https://<span data-content="tenant-id"></span>.<span data-content="cluster-id"></span>.cloud.thethings.industries:443
    </code>
    </p>

> - `--current-gateway-key`: The API Key by the gateway currently to connect to the CUPS Server.

Reviewer: ```suggestion
- `--current-gateway-key`: The API Key that is currently used by the gateway to connect to the CUPS Server.
```

> 
> {{< cli-only >}}

Reviewer: ```suggestion
```
